addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -58,6 +58,9 @@ $-editor-messages-margin-x: 2%;
             min-width: 0;
         }
     }
+    &[data-oe-type] {
+        display: inline-block;
+    }
```

Comment:
I checked it again.

> Either another container or another css rule

You were right; the problem is with the container.

It happens when a inline element and its parent element with contenteditable false is nested within an anchor tag.
I was able to replicate the same behavior on [jsfiddle](https://jsfiddle.net/uL6t4f7y/).

Reply:
Great find, thanks! I have the same behavior if I remove the contenteditable false parent, so with only the link and the small. Do you observe the same behavior?

Reply:
> Do you observe the same behavior?

Yes its the same without contenteditable false parent too.

Reply:
Or maybe should I fix it in js where I could specifically target `data-oe-fields` with closest `a` tag and has display `inline`?

Reply:
That would work too, yes. In `fillEmpty`, if you can detect that this is the special case in which putting a zwsp is not enough to make it possible to edit it (since it's the goal of fillEmpty), then you can add the display. Don't forget to add a comment because it's quite a weird case.